### PR TITLE
feat: initial working version

### PR DESCRIPTION
**Pull request by HarikrishnanBalagopal**
Gets information about releases:
current version, next version and
next next version.

Signed-off-by: Harikrishnan Balagopal <harikrishmenon@gmail.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -38,7 +42,10 @@ bower_components
 build/Release
 
 # Dependency directories
+<<<<<<< HEAD
```
Comment (ashokponkumar) [inline .gitignore:45]: Seems like a merge conflict?

File: .gitignore [hunk 2]
```diff
@@ -53,12 +60,15 @@ typings/
 # Optional eslint cache
 .eslintcache
 
+<<<<<<< HEAD
```
Comment (ashokponkumar) [inline .gitignore:63]: We need to resolve this merge confict.

**Review by ashokponkumar — CHANGES_REQUESTED** [review]
Do we need the dist folder to be checked in?

Is package-lock.json generally this big a file?

File: .gitignore (hunk 2 above)
Comment (HarikrishnanBalagopal) [reply .gitignore]: done

File: .gitignore (hunk 1 above)
Comment (HarikrishnanBalagopal) [reply .gitignore]: done

**Comment by HarikrishnanBalagopal** [comment]
> Do we need the dist folder to be checked in?
> 
> Is package-lock.json generally this big a file?

Yeah the dist folder contains the final bundled js file that is being run

**Review by ashokponkumar — APPROVED** [review]

**Pull request merged**